developer-portal/excluded-docs/_TEMPLATE-PLATFORM_NAME-use-fast-headless-checkout.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Fast PLATFORM_NAME Docs: Fast Headless Checkout with PLATFORM_NAME"
+description: Fast Headless Checkout with PLATFORM_NAME
+keywords: Fast Headless Checkout PLATFORM_NAME
+exclude: true
+---
+
+### Use the Fast Checkout Button - The Headless Approach
```

Comment:
nit: do we need these to start at h3? the header text starts to get very small on this page 😓

Reply:
fixed